### PR TITLE
feat: add basic sipmle  componets

### PR DESCRIPTION
**Pull request by rhryhoryk**

### CODE REVIEW

File: src/app/app.component.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
+app-header {
+  padding: 0.8%;
```
Comment (viento006) [inline src/app/app.component.css:2]: better use px here

File: src/app/header/header.component.css [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  width: 30%;
+  margin: 1vw auto;
+  padding: 2% 10px;
+
+  overflow: hidden;
+
+  border-width: 0;
+  outline: none;
+  border-radius: 2px;
+  box-shadow: 0 1px 4px #00000099;
```
Comment (viento006) [inline src/app/header/header.component.css:18]: put all colors in variables

File: src/app/header/header.component.css [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+  text-align: center;
+  text-decoration: none;
+
+  background-color: var(--active);
+  color: var(--text);
+
+  transition: background-color .3s;
+}
+
```
Comment (viento006) [inline src/app/header/header.component.css:28]: unnnessesary space

File: src/app/header/header.component.css [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+.anchor-btn:before {
+  content: "";
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  display: block;
+  width: 0;
+  padding-top: 0;
+  background-color: #ecf0f14d;
```
Comment (viento006) [inline src/app/header/header.component.css:43]: put all colors in variables

File: src/app/header/header.component.html [hunk 5]
```diff
@@ -0,0 +1,5 @@
+<div class="logo">logo</div>
+<nav class="wrapper--flex">
+  <a class="anchor-btn" *ngFor="let btn of buttons" routerLink="{{btn.url}}">{{btn.name}}</a>
```
Comment (viento006) [inline src/app/header/header.component.html:3]: prefer using full words like button instead btn

File: src/app/header/header.component.ts [hunk 6]
```diff
@@ -0,0 +1,19 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-header',
+  templateUrl: './header.component.html',
+  styleUrls: ['./header.component.css']
+})
+export class HeaderComponent implements OnInit {
+  buttons = [
+    {name: 'movies', url: '/movies'},
```
Comment (viento006) [inline src/app/header/header.component.ts:10]: spaces inside brackets

File: src/styles.css [hunk 7]
```diff
@@ -1 +1,44 @@
...
+}
+
+app-root {
+  display: flex;
+  flex-direction: column;
+  min-height: 100%;
+}
+
+:root {
+  --active:      #e8c547;
```
Comment (viento006) [inline src/styles.css:25]: extract colors in a separate file. also you can use scss variables, but it's not mandatory

File: src/styles.css [hunk 8]
```diff
@@ -1 +1,44 @@
 /* You can add global styles to this file, and also import other style files */
+* {
```
Comment (viento006) [inline src/styles.css:2]: you can use reset.css or normalize.css for these purposes

**Review by viento006 — CHANGES_REQUESTED** [review]

File: src/styles.css (hunk 8 above)
Comment (rhryhoryk) [reply src/styles.css:2]: is it really necessary?

File: src/styles.css (hunk 7 above)
Comment (rhryhoryk) [reply src/styles.css:25]: I just like pure css=)

**Review by viento006 — APPROVED** [review]

**Pull request merged**